README.md

```diff
@@ -82,7 +84,7 @@ Papers
 
 ## License
 
-Copyright © 2020-2022 Informal Systems
+Copyright © 2020-2023 Informal Systems
```

Comment:
```suggestion
Copyright © 2020- Informal Systems and contributors
```

No need for an end date apparently. And, from what I understand, copyright belongs to whoever submitted the PR, so it's probably just worth mentioning contributors here.

Reply:
If you don't have a date range you might as well drop the hyphen too

```suggestion
Copyright © 2020 Informal Systems and contributors
```

Reply:
ddfc055